time-it-macros/src/lib.rs

```diff
@@ -19,13 +16,14 @@ pub fn time_it(to_time: TokenStream) -> TokenStream {
     quote! {
         let #start_ident = std::time::Instant::now();
         #to_time
-        println!("Operation took {}ms", #start_ident.elapsed().as_millis());
+        let duration = #start_ident.elapsed();
+        let config = time_it::get_config();
+        let action = config.action.clone();
+        action(duration);
```

Comment:
nit: could shorten this to `time_it::action(duration)` which just does these 3 lines under the hood

Reply:
would also remove need for arc clone because the static action function could get config directly and just use ref

Reply:
Added the action method for a better interface but observed no change in latency. Will keep the Arc here just in case we are adding more fields into the config and want to pass it out.